testscommon/redisClientStub.go

```diff
@@ -51,6 +51,11 @@ func (r *RedisClientStub) SetExpireIfNotExists(ctx context.Context, key string,
 	return false, nil
 }
 
+// SetPersist -
+func (r *RedisClientStub) SetPersist(ctx context.Context, key string) (bool, error) {
+	return false, nil
```

Comment:
```suggestion
	return true, nil
```

in order to avoid the log in tests

Reply:
fixed